internal/lm/fabric.go

```diff
@@ -142,18 +143,16 @@ func getImexDomainID(r io.Reader) (string, error) {
 }
 
 func generateContentUUID(seed string) string {
```

Comment:
Before changing this, we should add some tests for expected outputs so that we understand the implications of these changes.

Reply:
https://github.com/NVIDIA/k8s-device-plugin/pull/999

Reply:
Thanks